rups/src/cmd.rs

```diff
@@ -44,6 +49,8 @@ impl<'a> Command<'a> {
             Self::SetUsername(username) => vec![username],
             Self::SetPassword(password) => vec![password],
             Self::List(query) => query.to_vec(),
+            #[cfg(feature = "write")]
+            Self::Run(cmd, param) => vec![cmd, param.unwrap_or("")],
```

Comment:
It should emit a Vec of length 1 if there is no param; something like

```suggestion
            Self::Run(cmd, param) => param.map(|param| vec![cmd, param]).unwrap_or_else(|| vec![cmd]),
```

Reply:
Makes sense. Made the change.

Reply:
Ha. Messed up the commit message :). I put some words in `back-tics` and the shell evaluated it. 🤦 Hopefully you can squash the commits when merging!